docs/api/security/crypto.md

```diff
@@ -0,0 +1,49 @@
+## Crypto
```

Comment:
Query: Should this be "Mbed Crypto" for proper branding?

Reply:
TLS used only "TLS", so I did the same. I'll add Mbed.